### ISSUE TITLE
Any way to limit used bandwith?

### ISSUE THREAD

**Issue by jpmath54** [gilnobrega/farmr#155]
First of all, I would like to throw some congrats about the work done here, really a lifesaver.

Now my concern :
I'm farming XCH + 9 forks.
When I add Farmr, I see a real drop in my available (and also not really powerfull) DSL bandwithn and capacity to log remotely.

I'm thinking about monitoring only XCH and only one fork, but I really like the idea to have all of them at a glance on my smartphone...

Is there any mechanism to limit the BW consumption of farmr, maybe a way to reduce reports frequency or log analysis?
Sorry if it is already documented, Didn't found out in the github documentation.

**Comment by gilnobrega** [comment]
Yes, you can clone the repository, change this constant to the value you want in the source code https://github.com/gilnobrega/farmr/blob/e617ad810255b6cd75b711c1dc335addc9d4cdcf/farmr.dart#L33

Then you can compile farmr by following these instructions https://github.com/gilnobrega/farmr/blob/main/docs/installation-source.md

Keep in mind that any value above 30 minutes will trigger the "Device is offline" notification. This is handled by the server and there is nothing you can do about it at the moment.

The server code is being rewritten to allow people to host their own servers and customise these values.

**Comment by gilnobrega** [comment]
Performance issues with log parsing are being addressed in e77db97c0f54ce3b0236c5e67f11c09c68a9e711
It should perform much better in 1.7.8

**Comment by gilnobrega** [comment]
Addressed in [v1.8.0.0](https://github.com/gilnobrega/farmr/releases/tag/v1.8.0.0) 
3a6040120fd09b66f56d8144c545139ec12316c8
